torba/basedatabase.py

```diff
@@ -475,11 +480,10 @@ def add_keys(self, account, chain, keys):
         ) + ', '.join(['(?, ?, ?, ?, ?)'] * len(keys))
         values = []
         for position, pubkey in keys:
-            values.append(pubkey.address)
-            values.append(account.public_key.address)
-            values.append(chain)
-            values.append(position)
-            values.append(sqlite3.Binary(pubkey.pubkey_bytes))
+            values.extend([
+                pubkey.address, account.public_key.address, chain, position,
+                sqlite3.Binary(pubkey.pubkey_bytes)
+            ])
```

Comment:
If you're going to micro optimize :-P might as well make this a tuple instead of a list.